README.md

```diff
@@ -112,6 +112,28 @@ Examples:
 > Clears all entries from the address book.  
 Format: `clear`  
 
+#### Printing a sorted list of the people in the addressbook: `sort`
+> prints all the people in the list in an alphabetically sorted manner, but does not sort in the internal data list.
```

Comment:
good to see you have updated the documentation when you added new functionality/features :+1: